packages/api/src/modules/users/router.ts

```diff
@@ -133,4 +144,21 @@ router.post(
     }
 );
 
+router.post(
+    "/updatedescription",
+    requireAuthenticated(),
+    validateSchema(UpdateCountry, "body"),
+    async (ctx, next) => {
+        const { description } = ctx.request.body;
+        const { user } = ctx.session!;
+
+        await updateDescription("id", user, description);
+
+        ctx.status = 201;
```

Comment:
```suggestion
        ctx.status = 200;
```

201 is for when you create something. Use 200.